Projects/DesignSystem/Sources/TextField/SearchTextField.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  SearchTextField.swift
+//  DesignSystem
+//
+//  Created by Chanhee Jeong on 2022/10/25.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+import UIKit
+import SnapKit
+
+public final class SearchTextField: UIView {
+    public var placeholder: String = "장소"
+    
+    public let textField = UITextField()
+    private let leftImage = UIImageView()
+    private let backgroundView = UIView()
+    private let cornerRadius: CGFloat = 45/2
+    
+    public init() {
+        super.init(frame: .zero)
+        configureUI()
+        createLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension SearchTextField {
+    private func configureUI() {
+        textField.placeholder = placeholder
+        textField.clearButtonMode = .always
+        textField.returnKeyType = .go
```

Comment:
p2;

이거 .search가 있네요! 이게 나을듯요

```suggestion
        textField.returnKeyType = .search
```

Reply:
✨ 반영